eng/Versions.props

```diff
@@ -33,7 +33,7 @@
     <CodeStyleAnalyzerVersion>3.8.0</CodeStyleAnalyzerVersion>
     <VisualStudioEditorPackagesVersion>16.8.181</VisualStudioEditorPackagesVersion>
     <ILToolsPackageVersion>5.0.0-alpha1.19409.1</ILToolsPackageVersion>
-    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>16.9.43</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
+    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>16.9.150</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
```

Comment:
Out of curiosity do you all manually update ngen information at VS insertion time or do you somehow automate it?

Reply:
I'm pretty sure it's automated, though I'm not exactly sure where. @RikkiGibson would you know?

Reply:
automated part of the commit the insertion tool creates, see https://github.com/dotnet/roslyn/blob/d28b17a3c19ac6a6d41d214dddd7cdd0bf75a169/src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj#L102

Reply:
😮 we'll need to sync up on how to utilize this. We do it manually today and hollllly hell it's a pain.